src/Conversations/ConversationFilters.php

```diff
@@ -98,10 +95,7 @@ public function getParams(): array
     }
 
     /**
-     * @param int   $id
      * @param mixed $value
```

Comment:
Just for my own reference, is this `@param` kept because `$value` is _not_ type-hinted?

Reply:
Yep!